docs/docs/tutorials/creating-cds-dashboard.md

```diff
@@ -1,20 +1,17 @@
----
-id: creating-cds-dashboard
-title: Create and Deploy a dashboard
----
+______________________________________________________________________
+
+## id: creating-cds-dashboard title: Create and Deploy a dashboard
 
 # Create a dashboard with `Panel` and deploy it with `CDS Dashboards`
 
 ## What is a dashboard?
 
 Analyzing data provides us with insights. A dashboard stitches these insights together into a meaningful story.
 
-Different tools and technologies can be used to create and deploy a dashboard. In this tutorial
-we will be building a simple dashboard on Nebari which can be shared with other users and groups via
-[ContainDS Dashboards](https://cdsdashboards.readthedocs.io/en/stable/) (CDS Dashboards).
+Different tools and technologies can be used to create and deploy a dashboard. In this tutorial we will be building a simple dashboard on Nebari which can be shared with other
+users and groups via [ContainDS Dashboards](https://cdsdashboards.readthedocs.io/en/stable/) (CDS Dashboards).
```

Comment:
which hook is unwrapping these lines? That's very frustrating as a writer. mdformat is set to 180. That needs to be changed to 120. That may be expanding to 120 in addition to cutting things off at 120.